src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -232,13 +233,23 @@ export class MeetingManager implements AudioVideoObserver {
     this.meetingRegion = meetingInfo.MediaRegion;
     this.meetingId = this.configuration.meetingId;
 
-    if (eventReporter) {
-      this.eventReporter = eventReporter;
+    if (eventController) {
+      if(this._eventController)
+      {
+        this._eventController.removeObserver(this.eventDidReceiveRef);
+      }
+      this._eventController = eventController;
+      this._eventController.addObserver(this.eventDidReceiveRef);
     }
 
     await this.initializeMeetingSession(this.configuration, deviceLabels);
   }
 
+  componentWillUnmount() {
```

Comment:
We cannot do this here, since MeetingManager is not a component 😓 but just a JavaScript class and not a React component.